content/client-server-api/modules/content_repo.md

```diff
@@ -19,6 +19,12 @@ When serving content, the server SHOULD provide a
 `Content-Security-Policy` header. The recommended policy is
 `sandbox; default-src 'none'; script-src 'none'; plugin-types application/pdf; style-src 'unsafe-inline'; object-src 'self';`.
 
+{{% added-in v="1.4" %}}
```

Comment:
I don't find it clear which part was added: was it the bit before (about C-S-P) or the bit after the '[Added in]' text?

Reply:
Precedent is always the text after. Suggestions welcome for how to fix it, though I will note that inline annotations are basically impossible.

Reply:
With an inline annotation, like https://github.com/matrix-org/matrix-spec/blame/main/content/client-server-api/_index.md#L363, or frankly, just:

``` 
**[Added in `v1.4`]**: foo bar
```

without any templating magic

Reply:
The inline annotation doesn't work in paragraphs for annoying and complicated reasons - it works on lists though.

I would prefer it if we always used the template for find-and-replace reasons, or if we decide to change the format down the line. I'm willing to codify this somewhere, if needed.

Reply:
well, I don't know. Can we stick it in a box of some kind? or use a bullet list, if that's enough to make the annotation work (one bullet for each of CSP and CORP).

I feel like we need to do *something* here: it's very unclear as it is.

Reply:
ftr, the inline issue is this: https://github.com/matrix-org/matrix-spec/issues/1204

I've fixed the inline-ness using a somewhat ugly hack, but it's probably fine enough for now.